man/systemd.network.xml

```diff
@@ -960,6 +960,16 @@
         </varlistentry>
 
         <varlistentry>
+          <term><varname>UserClass=</varname></term>
+          <listitem>
+            <para>A DHCPv4 client can use UserClass option to identify the type or category of user or applications
+            it represents. The information contained in this option is an string object that represents the user class
+            of which the client is a member. Each class sets an identifying string of information to be used by the DHCP
+            service to classify identifying clients. Takes a whitespace-separated list of user class information.</para>
```

Comment:
This setting should follow our usual logic, and hence we should ensure that each time the setting is used a new entry is added to the list of user class identifiers, except if the empty string is assigned, in which case the whole list is reset. We generally do it this way with the options that take lists of strings, and so we should do it here too.